subsys/cpu_load/Kconfig

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2025 Analog Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig CPU_LOAD_SUBSYS
```

Comment:
SUBSYS should not be in the configs name, it doesn't really convey much information, CPU_LOAD_INFO or CPU_USAGE even would be more descriptive :)